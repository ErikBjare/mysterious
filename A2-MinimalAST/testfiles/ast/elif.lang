int main() {
    if(1) {
        main();
    } else if(2) {
        main();
    } else {
        int a = 5;
    }
}