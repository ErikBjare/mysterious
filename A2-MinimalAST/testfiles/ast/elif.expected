Program
  List
    Function
      IdDecl ID="main" TYPE="int"
      List
      List
        IfStmt
          IfBlock
            Literal LITERAL="1"
            List
              ExpressionStmt
                FunctionCall
                  IdUse ID="main"
                  List
          List
            IfBlock
              Literal LITERAL="2"
              List
                ExpressionStmt
                  FunctionCall
                    IdUse ID="main"
                    List
          Opt
            ElseBlock
              List
                IdStmt
                  IdDecl ID="a" TYPE="int"
                  Opt
                    Literal LITERAL="5"

